### PR TITLE
Briefer changelog format

### PR DESCRIPTION
**Pull request by jwnimmer-tri**
Update the format to have more terseness and organization.
Add more content.

### CODE REVIEW

**Comment by jwnimmer-tri** [comment]
@liangfok PTAL for first-pass review.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,25 +1,58 @@
...
+[//]: # "New functionality or APIs."
+### Added
+
+ - None
+
+[//]: # "Altered functionality or APIs."
+### Changed
+
+ - [#1992][] Matlab tests must now be explicitly listed in CMakeLists.
+ - [#1970][] The drake pod Makefile now includes the `install` action.
```
Comment (liangfok) [inline CHANGELOG.md]: cap: drake --> Drake

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,25 +1,58 @@
...
+ - [#1953][] Replace valuecheckMatrix() with CompareMatrices().
```
Comment (liangfok) [inline CHANGELOG.md]: Surround method names with back ticks.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,25 +1,58 @@
...
-### ([@liangfok][], [#1953][]) - Replacement of valuecheckMatrix() with CompareMatrices()
+ - None
 
-Method `valuecheckMatrix()` in `drake/util/testUtil.h` was removed and replaced with `drake::util::CompareMatrices()` in `drake/util/eigen_matrix_compare.h`.
+[//]: # "Smaller bug fixes.  No API changes."
+### Fixes
 
-Unit tests that originally contained
+ - [#2008][] `Drake::simulate(4-arg)` actually compiles again now.
+ - [#1990][] Windows builds that include bullet are working again.
```
Comment (liangfok) [inline CHANGELOG.md]: cap: bullet --> Bullet

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,25 +1,58 @@
...
+ - [#1985][] runQuadrotorDynamics no longer asserts realtime performance when run under ctest.
```
Comment (liangfok) [inline CHANGELOG.md]: Should `runQuadrotorDynamics` be surrounded in back ticks?

Comment (liangfok) [inline CHANGELOG.md]: Consider adding the word "violation" after "performance".

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,25 +1,58 @@
...
+ - [#1975][] Fix coordinate frame error in rigid body collisions.
+ - (Assorted) Non-functional fixes for cpplint compliance.
```
Comment (liangfok) [inline CHANGELOG.md]: Should `cpplint` be surrounded by back ticks?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,25 +1,58 @@
...
 
-    valuecheckMatrix(matrix1, matrix2, tolerance);
+Note that not all changes since v0.9.11 have been captured in the above.
```
Comment (liangfok) [inline CHANGELOG.md]: Remove "in the" or add "lists" after "above".

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,25 +1,58 @@
...
+Only changes since approximately 2016-04-01 have been noted.
```
Comment (liangfok) [inline CHANGELOG.md]: Consider changing "have been" to be "are".

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,25 +1,58 @@
...
-should be modified to use Google Test and the following line:
+v0.9.11 (2015-10-08)
+--------------------
 
-    EXPECT_TRUE(drake::util::CompareMatrices(matrix1, matrix2, tolerance, MatrixCompareType::absolute));
+Changes in version v0.9.11 and before are not provided.
 
-<!--- The following link definition list is generated by PimpMyChangelog --->
+[//]: # "You can use PimpMyChangelog to auto-update this list."
+[//]: # "https://github.com/pcreux/pimpmychangelog."
```
Comment (liangfok) [inline CHANGELOG.md]: Is that an extraneous period?

**Comment by liangfok** [comment]
Done with my first pass review.

File: CHANGELOG.md (hunk 3 above)
Comment (RussTedrake) [inline CHANGELOG.md]: I think the last three are probably all too small to inckude in the changelog, no?

File: CHANGELOG.md (hunk 4 above)
Comment (RussTedrake) [inline CHANGELOG.md]: Too small

**Comment by jwnimmer-tri** [comment]
Over to @david-german-tri for second review and merge.

**Comment by david-german-tri** [comment]
Test: does just commenting on a thread restart CI?

**Pull request merged**